testing/avara/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: shrizza <shrizza@gmail.com>
+# Maintainer: shrizza <shrizza@gmail.com>
+pkgname="avara"
+pkgver="0_git20191006"
+pkgrel=0
+pkgdesc="Port of the original 1996 Mac OS game from Ambrosia Software"
+url="https://github.com/avaraline/Avara"
+arch="all"
+license="MIT"
+depends=""
+makedepends="clang clang-dev directfb-dev glu-dev sdl2_net-dev sdl2_mixer-dev sdl2"
+options="!check"
+install=""
+subpackages="$pkgname-doc"
+source="avara https://github.com/avaraline/Avara/archive/b8700db.zip"
```

Comment:
Make a `_sha` variable, which you can use both here and in the builddir, so this is:
It's `$pkgname-$pkgver.zip::https://github.com/avaraline/Avara/archive/$_sha.zip`